### PR TITLE
Work Section and About Me Section

### PR DESCRIPTION
**Pull request by Sonikak004**
### In this Branch I have:

- created _work section_ with background images using **HTML** and **CSS**
- created _about me section_ with illustration using **HTML** and **CSS**
- Used **Flex Box** and **Grid Layout** 
- _Linters_ checked 

> Tried to replicate _**Figma Design Template**_

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,4 +63,191 @@ <h2 class="profession">I'am a Software Developer</h2>
...
+    <!-- recent work submit button -->
+
+    <button class="btn" type="button">
+      <a class="button-text">See Project</a>      
+    </button>
+  </section> 
+
+     <!-- professional art cards section -->
+
+     <section class="professional-art">
```
Comment (bobb-Rob) [inline index.html:106]: - [ ] This section is expected to use `display: grid` to lay out each project. Please implement this core project requirement.  The advantage of using Grid display is to make your portfolio page responsive.

File: index.html [hunk 2]
```diff
@@ -63,4 +63,191 @@ <h2 class="profession">I'am a Software Developer</h2>
...
+      <!-- First professional skill card -->
+
+      <div id="pro-images-1" class="professional-art-text"> Profesional Art Printing Data
```
Comment (bobb-Rob) [inline index.html:109]: - [ ] Per the project requirement, please use one of the `hx` tags i.e `h2`, `h3` etc. for the title of each project card. **Profesional Art Printing Data**

File: index.html [hunk 3]
```diff
@@ -63,4 +63,191 @@ <h2 class="profession">I'am a Software Developer</h2>
...
+        <!-- skill button in professional sectiion -->
+      <ul>
+        <li class="card1">html</li>
+        <li class="card2">bootsrap</li>
+        <li class="card3">Ruby</li>
+      </ul> 
+
+      <button class="button2" type="button">
+        <a class="button-text2" href="#"> See Project </a>  
+    </button>
```
Comment (bobb-Rob) [inline index.html:120]: - [ ] Please you can either use a button or an anchor tag but not both of them at the same time.

File: index.html [hunk 4]
```diff
@@ -63,4 +63,191 @@ <h2 class="profession">I'am a Software Developer</h2>
...
+        <br> <br>
+
+        <!-- skills list -->
+        <div class="about-heading-skills"> <img src="./images/circle.png" alt="circle logo" class="circle-img"> Skills
+        </div>
+          <h3 class="about-head-text">Codekit</h3>
+          <h3 class="about-head-text">Github</h3>
+          <h3 class="about-head-text">Codepen</h3>
+          <h3 class="about-head-text">Gitlab</h3>
+          <h3 class="about-head-text">Terminal</h3>
```
Comment (bobb-Rob) [inline index.html:252]: - [ ] To improve your page accessibility, the technology categories **("Languages", "Frameworks" and "Skills")** should be in a `ul`, each of them wrapped in a `li`. Also, the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`.  Kindly implement this.

File: index.html [hunk 5]
```diff
@@ -63,4 +63,191 @@ <h2 class="profession">I'am a Software Developer</h2>
...
+      <!-- about me section -->
+
+     <section class="about-me">
+      <h2 class="about-me-text">About me</h2>
+      <p class="about-me-para">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+
+      <!-- get my resume button -->
+      <button class="button3" type="button">
+        <a class="button-text3" href="#"> Get My Resume </a>  
+    </button>
```
Comment (bobb-Rob) [inline index.html:218]: - [ ] This section has a background image as seen below in the Figma design template. Please do not use `img` tag but use CSS `background-image` property to add this background illustration to the section wrapper.


Your design | Figma Design
---- | ----
image | image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @Sonikak004 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by NahnahAJ — APPROVED** [review]
Hi @Sonikak004 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
All linters are well set up and checks passed :heavy_check_mark:
Pull request title is well done :heavy_check_mark:
Pull request summary is descriptive :heavy_check_mark:
Meaningful commit messages :heavy_check_mark:
Nice design :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**